attributes/default.rb

```diff
@@ -1,5 +1,11 @@
 
 # Override cookbook attributes to install Java 7
-override['java']['openjdk_packages'] = [
-  "openjdk-7-jdk", "openjdk-7-jre-headless"
-]
+#override['java']['openjdk_packages'] = [
+#  "openjdk-7-jdk", "openjdk-7-jre-headless"
+#]
```

Comment:
Awesome! If you would be willing to commit those two lines and delete the comment, that would be great!